spec/hand_rankings/hand_entities/high_card_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'poker_hands/hand_rankings/find_high_card'
+require 'poker_hands/hand_rankings/hand_entities/high_card'
+require 'poker_hands/card'
+require 'spec_helper'
+
+
+RSpec.describe PokerHands::Entities::HighCard do
+  describe '#initialize' do
```

Comment:
We generally don't test `#initialize`. If our initialization step has side-effects/public behaviour we should reconsider what our object initialization really needs to do. I think I'd rather see an isolated test for `type`. For example:

```
describe 'type' do
  # With an appropriate subject

  it { is_expected.to eq('high card') }
end